net-misc/zerotier/zerotier-1.2.8.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic toolchain-funcs systemd
+
+HOMEPAGE="https://www.zerotier.com/"
+DESCRIPTION="A software-based managed Ethernet switch for planet Earth"
+SRC_URI="https://github.com/zerotier/ZeroTierOne/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+S="${WORKDIR}/ZeroTierOne-${PV}"
+
+RDEPEND="
+	net-libs/miniupnpc
+	net-libs/libnatpmp
+	dev-libs/json-glib
+	net-libs/http-parser"
+
+DEPEND="${RDEPEND}
+	>=sys-devel/gcc-4.9.3"
```

Comment:
Any specific reason to depend on gcc?

Reply:
This is a requirement from upstream.

See https://github.com/zerotier/ZeroTierOne/issues/697

Reply:
no matter here, gcc belongs system gentoo set, the packages from this sset should almost never be the dependencies for regular ebuilds, morover there's obivious a gcc versuib that `>=4.9.3` in gentoo

Reply:
I will pull it out then. Thanks!